### ISSUE TITLE
Request: Select All & Delete

### ISSUE THREAD

**Issue by dannyboyLP** [teresa-ou/inboxy#106]
The Archive button is great, but it would be really useful to be able to bulk delete all in a bundle, and also to select all messages in a bundle for a different action.

**Comment by kidrazzledazzle** [comment]
Yes, definitely agree. I receive emails from all of our OUs, and I bundle them into one big group. I really don't need to read them all, and I definitely don't need to save them. It would be nice if there was a bulk delete option. 

BY THE WAY. I -love- this extension. I've been so sad ever since Inbox was cancelled, and you've brought light back into my dark and sad world. :D

**Comment by benoror** [comment]
Hi! (Replying from the maintained fork side.)

**Select all** on a bundle shipped in **Inbundly** v4.2.0 (checkbox on the row). **Archive-all** is on by default. A dedicated **Delete all** button is not in yet; after select-all you can use Gmail's Delete, or open a feature request for a one-click delete control (same ask as several older threads / PR #71).

Chrome: https://chromewebstore.google.com/detail/inbundly/pbfjicjmcpogjlbpljebhhgkgfhbdcga
Firefox: https://addons.mozilla.org/firefox/addon/inbundly/
Site: https://inbundly.com/
Issues: https://github.com/benoror/inbundly/issues
